service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -620,19 +627,20 @@ func (t *transferQueueStandbyTaskExecutor) fetchHistoryFromRemote(
 			0,
 			0,
 		)
-	} else {
-		err = serviceerror.NewInternal(
-			"transferQueueStandbyProcessor encountered empty historyResendInfo",
-		)
 	}
 
 	if err != nil {
+		if _, isNotFound := err.(*serviceerror.NamespaceNotFound); isNotFound {
```

Comment:
nit: can we define a util function for this check? maybe in the `common` package?

Reply:
no. I am against this. go codders should suffer from all go features!